src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/RicercaNotificheMittentePagoPATest.java

```diff
@@ -28,7 +33,7 @@
 
 public class RicercaNotificheMittentePagoPATest extends BasePage {
 
-    private static final Logger logger = LoggerFactory.getLogger("NotificaMittenteTest");
+    private static final Logger logger = LoggerFactory.getLogger("RicercaNotificheMittentePagoPATest");
```

Comment:
come best practice meglio usare direttamente la classe nel getLogger
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(RicercaNotificheMittentePagoPATest.class);
```